### DISCUSSION TITLE
blog/2022/09/26/guide-visualizing-types-posteriors/

### DISCUSSION THREAD

**Discussion by giscus** [andrewheiss/ath-quarto#69] [Blog comments]
# blog/2022/09/26/guide-visualizing-types-posteriors/

A guide to different types of Bayesian posterior distributions and the nuances of posterior_predict, posterior_epred, and posterior_linpred

https://www.andrewheiss.com/blog/2022/09/26/guide-visualizing-types-posteriors/

**Comment by kjwalter89** [comment]
Thank you for this! If you were generating predictions m a zero-one inflated beta regression (i.e., family = zero_one_inflated_beta() in brms) would you use posterior_predict?
